### DISCUSSION TITLE
basic test error: expected 51 to equal 50 JS

### DISCUSSION THREAD

**Discussion by JuanCarlosAld2** [codewars/codewars.com#2857] [Q&A]
hi everybody, I do this exercise and my test it's ok, but when click ATTEMPT boton the Output print this error:  "expected 51 to equal 50". The lenguage that i use is JavaScript, can you help me please? I've all day train do this.
TEST
![image](https://user-images.githubusercontent.com/111910306/200947417-c95983c1-d094-4bda-a294-c558a1adf1aa.png)
ATTEMPT
![image](https://user-images.githubusercontent.com/111910306/200947591-6ab7ef2e-762d-422a-b4ae-6efbfb67b5ec.png)
Thanks!

**Comment by Blind4Basics** [comment]
you can see the inputs for the failing test by adding `console.log(...)` at the first line inside your function. You'll see them in the output panel, just above the failing assertion.
